jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9VirtualMachine.java

```diff
@@ -385,7 +385,7 @@ private void tryAttachTarget(int timeout) throws IOException,
 
 				targetServer = new ServerSocket(0); /* select a free port */
 				portNumber = Integer.valueOf(targetServer.getLocalPort());
-				String key = Integer.toHexString((IPC.getRandomNumber()));
+				String key = IPC.getRandomString();
```

Comment:
Is `IPC.getRandomNumber()` used anymore?  Can it be deleted?

Reply:
It's used in FileLock.java (ca. line 177) when dealing with stale files.